srcpkgs/nyx/template

```diff
@@ -1,14 +1,18 @@
 # Template file for 'nyx'
 pkgname=nyx
 version=2.1.0
-revision=5
+revision=6
 build_style=python3-module
-pycompile_module="nyx"
 hostmakedepends="python3-setuptools"
 depends="python3-setuptools python3-stem tor"
+checkdepends="ncurses-base python3-wheel python3-pip python3-stem"
 short_desc="Command-line status monitor for tor"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Frank Steinborn <steinex@nognu.de>"
 license="GPL-3.0-or-later"
 homepage="https://nyx.torproject.org/"
 distfiles="${PYPI_SITE}/n/${pkgname}/${pkgname}-${version}.tar.gz"
 checksum=88521488d1c9052e457b9e66498a4acfaaa3adf3adc5a199892632f129a5390b
+
+do_check() {
+	./run_tests.py
+}
```

Comment:
this doesn't fail when tests fail

Reply:
If you are using pytest, then you can try `--deselect` or patching them out.

Reply:
Sorry, I'm not able to bring this further than to disable tests altogether... I'm really no python test-ninja and this doesn't seem to use `pytest`.